Javascript(ES6)/README.md

```diff
@@ -3126,9 +3126,8 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
 **Tools**
 
   - Code Style Linters
-    + [ESlint](http://eslint.org/) - [Airbnb Style .eslintrc](https://github.com/airbnb/javascript/blob/master/linters/.eslintrc)
+    + [ESlint](http://eslint.org/) - [Making Sense Style .eslintrc](.eslintrc)
     + [JSHint](http://jshint.com/) - [Airbnb Style .jshintrc](https://github.com/airbnb/javascript/blob/master/linters/.jshintrc)
```

Comment:
I believe community is giving the support to ESlint. I would flavor to delete JSHint. Anyways is fine. Just want to cut down with the JS-world overwhelming.